__tests__/components/sidebar.test.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { render, fireEvent, waitFor } from '@testing-library/react-native';
+import { useRouter, usePathname } from 'expo-router';
+import Sidebar from '@/components/sidebar'; // Adjust the import path as needed
+
+// Mock the expo-router hooks
+jest.mock('expo-router', () => ({
+  useRouter: jest.fn(),
+  usePathname: jest.fn(),
+}));
+
+// Mock the SvgXml component
+jest.mock('react-native-svg', () => ({
+  SvgXml: 'SvgXml',
+}));
+
+describe('Sidebar', () => {
+  const mockOnClose = jest.fn();
+  const mockPush = jest.fn();
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    (useRouter as jest.Mock).mockReturnValue({ push: mockPush });
+    (usePathname as jest.Mock).mockReturnValue('/dashboard');
+  });
+
+  it('renders correctly', () => {
+    const { getByText } = render(<Sidebar onClose={mockOnClose} />);
+    expect(getByText('Dashboard')).toBeTruthy();
+    expect(getByText('Attendance')).toBeTruthy();
+    expect(getByText('Performance')).toBeTruthy();
+    expect(getByText('Notifications')).toBeTruthy();
```

Comment:
On the web version, we decided to remove the Notifications link from the sidebar because it's already available on the top navbar.